src/utils/TrezorSigner.ts

```diff
@@ -0,0 +1,429 @@
+import {
+  bitcoinZPubTypes,
+  DEFAULT_TREZOR_DOMAIN,
+  isTrezorInitialized,
+  setTrezorInitialized,
+  Signer,
+  syscoinSLIP44,
+  syscoinZPubTypes,
+} from '.';
```

Comment:
I'm guessing it maybe better to just export these as `utils/constants.ts` to avoid circular dependency.

Reply:
I agree